spyder_notebook/notebookplugin.py

```diff
@@ -386,9 +393,10 @@ def open_fileswitcher_dlg(self):
             self.fileswitcher_dlg.hide()
             self.fileswitcher_dlg.is_visible = False
             return
-        self.fileswitcher_dlg = FileSwitcher(self, self.tabwidget,
-                                             self.clients)
+        self.fileswitcher_dlg = FileSwitcher(self, self, self.tabwidget,

```

Comment:
Please remove this instance of the file switcher and its associated button. Given the changes in your Spyder PR, I think we don't need this anymore.